### PR TITLE
991 Mark QC Work item as read

### PR DESCRIPTION
**Pull request by wbyoko**

### CODE REVIEW

File: web-client/src/presenter/actions/EditDocketRecord/setQCWorkItemIdToMarkAsReadIfNeededAction.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ */
+export const setQCWorkItemIdToMarkAsReadIfNeededAction = ({ get, props }) => {
+  let workItemIdToMarkAsRead;
+  const caseDetail = get(state.caseDetail);
+  const { documentId } = props;
+
+  const initialDocument =
+    caseDetail.documents.find(entry => entry.documentId === documentId) || {};
+
+  const unreadQCcWorkItem = (initialDocument.workItems || []).find(
```
Comment (kkoskelin) [inline web-client/src/presenter/actions/EditDocketRecord/setQCWorkItemIdToMarkAsReadIfNeededAction.js]: `unreadQCcWorkItem` "Cc"?

**Review by kkoskelin — APPROVED** [review]
spelling of variable name could be corrected before someone else (:cough: @waldoj ) finds it

File: web-client/src/presenter/actions/EditDocketRecord/setQCWorkItemIdToMarkAsReadIfNeededAction.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    wi => wi.isQC && !wi.isRead,
```
Comment (sutt0n) [inline web-client/src/presenter/actions/EditDocketRecord/setQCWorkItemIdToMarkAsReadIfNeededAction.js:20]: can we spell out `workItem` here?

**Review by sutt0n — CHANGES_REQUESTED** [review]

File: web-client/src/presenter/actions/EditDocketRecord/setQCWorkItemIdToMarkAsReadIfNeededAction.js (hunk 2 above)
Comment (wbyoko) [reply web-client/src/presenter/actions/EditDocketRecord/setQCWorkItemIdToMarkAsReadIfNeededAction.js:20]: is there a reason to spell it out? For shorter functions like this I don't see the issue in having an obvious abbreviation as the variable name.

Comment (sutt0n) [reply web-client/src/presenter/actions/EditDocketRecord/setQCWorkItemIdToMarkAsReadIfNeededAction.js:20]: For the sake of clean-_er_ code. I agree that it's an obvious abbreviation, but I think for the sake of CA we should spell it out. Feel free to PM me about this. 😃

**Review by sutt0n — APPROVED** [review]

**Pull request merged**